src/field_theory/minimal_polynomial.lean

```diff
@@ -0,0 +1,228 @@
+/-
+Copyright (c) 2019 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Johan Commelin
+-/
+
+import ring_theory.integral_closure
+
+/-!
+# Minimal polynomials
+
+This file contains the utmost basics on minimal polynomials. If gives the definition in the setting
```

Comment:
Could you add more to the module doc, as per https://github.com/leanprover-community/mathlib/blob/master/docs/contribute/doc.md?

Reply:
There really isn't that much to say. The file is very short and straightforward. No notation, no interesting implementation details.

Reply:
I think I agree. The three main points (definition, irreducibility, existence) are covered in the module doc summary. The implementation is all smooth sailing.